content/en/docs/ops/ambient/getting-started/test.sh

```diff
@@ -76,3 +96,4 @@ cleanup_bookinfo_sample
 snip_uninstall_uninstall_1
 snip_uninstall_uninstall_2
 snip_uninstall_uninstall_3
+remove_gateway_api_crds "${KUBE_CONTEXTS[0]}"
```

Comment:
The @cleanup section should be only the not gateway-api cleanup and must be guarded because otherwise they will be called at the end of the test which is included in gtw_test.sh, instead of during the gtw_test.sh cleanup.
```
@cleanup
if [ "$GATEWAY_API" != "true" ]; then
    cleanup_bookinfo_sample
    snip_uninstall_uninstall_1
    snip_uninstall_uninstall_2
    snip_uninstall_uninstall_3
    snip_delete_crds
fi
```

Reply:
Gateway API CRDs are installed regardless of what APIs (Istio Classic|K8s Gateway) are being used for ingress ([xref](https://preliminary.istio.io/latest/docs/ops/ambient/getting-started#l7-authorization-policy)). `cleanup_bookinfo_sample` has separate cleanup steps based on `GATEWAY_API` and the other snips are required regardless of what APIs (Istio Classic|K8s Gateway) are being used.

Reply:
@frankbu thanks for the additional insight. It now makes sense to me.